<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMenu</class>
 <widget class="QDialog" name="AdminMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Меню администратора</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Меню</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Редактировать</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_admins">
          <property name="text">
           <string>Администраторы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_plans">
          <property name="text">
           <string>Планы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_dishes">
          <property name="text">
           <string>Блюда</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_recipes">
          <property name="text">
           <string>Рецепты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ingredients">
          <property name="text">
           <string>Ингредиенты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_storage">
          <property name="text">
           <string>Склад</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Формы</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_scripts">
          <property name="text">
           <string>Записи склада</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_balance">
          <property name="text">
           <string>Баланс ингредиентов</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
